src/main/resources/openapi/schemas/reportTitles.json

```diff
@@ -1,5 +1,5 @@
 {
-  "description": "report titles",
+  "description": "report titles (for both request and response)",
```

Comment:
Wait, does this mean that when I send manual updates to the title matching, I have to include `resultInfo`?

Reply:
No. It just means that this schema is used for both request and response. Should we define a schema for the request and another for the response? I don't mind..I was just lazy. resultInfo is not required in this schema obviously.

Reply:
Oh, course, it's optional. That's fine.